junction/proposals/tests/test_services.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+# Third Party Stuff
+from django.test import TestCase
+
+# Junction Stuff
+from junction.proposals.services import post_tweet_for_new_proposal
+
+
+class TestServices(TestCase):
+
+    def test_post_tweet_for_new_proposal(self):
+        proposal_name = 'test proposal'
+        proposal_url = 'http://in.pycon.org/2015/'
+        response = post_tweet_for_new_proposal(proposal_name, proposal_url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
this whole thing/file can also be reduced to this:

```
from junction.proposals.services import post_tweet_for_new_proposal

def test_post_tweet_for_new_proposal():
  proposal_name = 'test proposal'
  proposal_url = 'http://in.pycon.org/2015/'
  response = post_tweet_for_new_proposal(proposal_name, proposal_url)
  assert response.status_code == 200
```

 thanks to pytest (http://pytest.org) :blush: